target/linux/ath79/dts/ar7161_dlink_dir-825-b1.dts

```diff
@@ -0,0 +1,216 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	compatible = "dlink,dir-825-b1", "qca,ar7161";
+	model = "D-Link DIR825B1";
+
+	aliases {
+		led-status = &orange_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	extosc: ref {
+		compatible = "fixed-clock";
+		#clock-cells = <0>;
+		clock-output-names = "ref";
+		clock-frequency = <40000000>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_usb {
+			label = "d-link:blue:usb";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		orange_power: orange_power {
+			label = "d-link:orange:power";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
+
+		blue_power {
+			label = "d-link:blue:power";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		
+		blue_wps {
+			label = "d-link:blue:wps";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		orange_planet {
+			label = "d-link:orange:planet";
+			gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		blue_planet {
+			label = "d-link:blue:planet";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
+		wps {
+			label = "wps";
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	rtl8366s {
+		compatible = "realtek,rtl8366s";
+		gpio-sda = <&gpio 5 GPIO_ACTIVE_HIGH>;
+		gpio-sck = <&gpio 7 GPIO_ACTIVE_HIGH>;
+		
+		mdio-bus {
+			status = "okay";
+
+			phy-mask = <0x10>;
+
+			phy4: ethernet-phy@4 {
+				reg = <4>;
+				phy-mode = "rgmii";
+			};
+		};
+	};
+};
+
+&usb1 {
+	status = "okay";
+};
+
+&usb2 {
+	status = "okay";
+};
+
+&usb_phy {
+	status = "okay";
+};
+
+&pcie0 {
+	status = "okay";
+
+	ath9k0: wifi@168c,0029 {
+		compatible = "pci168c,0029";
+		reg = <0x0000 0 0 1 1>;
+		mtd-mac-address = <&caldata 0xffa0>;
+		qca,no-eeprom;
+		#gpio-cells = <2>;
+		gpio-controller;
```

Comment:
I think it is worth defining gpios in case someone wants to use them for some other indication.
But I need to make a node for them too.

Reply:
I am actually follwing the rule that exists in ar71xx platform. I think it is good to keep backward compatibility.

Reply:
> But I got syntax errors, probably on dash in "&ath9k-0"

Use an underline? Maybe something like ath9k_2g/ath9k_5g. It might be even better to use the chip name if multiple chip supported by the same driver are present. Not sure yet. Maybe someone else has an idea what makes sense here.

Reply:
It seems that `ath9k0` and `1` will do. There are two identical chips that are configured to be 2 and 5 GHz from eeprom.

Reply:
Maybe not quite identical ;-)

Reply:
> I am actually follwing the rule that exists in ar71xx platform. I think it is good to keep backward compatibility.

Please don't follow the rules existing in ar71xx. ar71xx is a mess and hopefully we are able to get rid of the mess with ath79.